PROVESFlightControllerReference/test/format_filesystem.py

```
@@ -0,0 +1,38 @@
+    """Fixture to check GDS is running"""
+
+    gds_working = False
+    timeout_time = time.time() + 30
+    while time.time() < timeout_time:
+        try:
+            fprime_test_api_session.send_and_assert_command(
+                command=f"{cmdDispatch}.CMD_NO_OP"
+            )
+            gds_working = True
+            break
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)